lib/__testfixtures__/created-instances.input.mjs

```diff
@@ -0,0 +1,8 @@
+const d = new Document();
```

Comment:
It's really unclear what is being tested by this. The file is named "created-instances" . I would expect that to have maybe

```js
(new Document()).createElement('div');
(new Document()).location;
(new Document()).length;

const d = new Document()
d.createElement('div');
d.location;
d.length // probably doesn't exist but some test of data property on created instance
```

I don't get why we then go on to make various accesses on `elt.attributes`.

Please add comments to explain what is the purpose of these other lines (in the context of created instances).

Also, perhaps you don't want the first set of things that don't assign to a variable. In that case maybe this file should use "assigned" rather than "created" in its name.